uk.co.compucorp.civicrm.tasksassignments/views/dashboard/key-dates.html

```diff
@@ -84,6 +84,7 @@ <h3 class="panel-title pull-left" ng-bind="(filterParams.date && filterParams.da
           </div>
         </div>
       </div>
+      <br/>
```

Comment:
Why do we have a `<br/>` here?

Reply:
My bad! thanks for catching, I think, I left it while debugging.